menu.txt

```diff
@@ -1,3 +1,3 @@
 dosa
-karam 
-oil
+extra karam 
+malasa dosa
```

Comment:
any paritucular reason to add the masala ?

Reply:
yes to become more tasty